<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="first.local" failures="1" tests="2" name="Tutorialspoint" time="20.519" errors="0" timestamp="2024-02-07T21:25:29 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.979"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="12.027">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.htm] but found [https://www.saucedemo.com/inventory.html]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.htm] but found [https://www.saucedemo.com/inventory.html]
at stepDef.productScreenTest.return_back_to_product_page(productScreenTest.java:46)
at âœ½.return back to product page(file:///Users/varadharajan/git/Sauce/SauceLabDemo/src/main/java/sauce/feature/productsPage.feature:20)
... Removed 6 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Tutorialspoint -->
